spec.html

```diff
@@ -26304,14 +26304,14 @@ <h1>Runtime Semantics: CreateDynamicFunction ( _constructor_, _newTarget_, _kind
               1. Let _prototype_ be OrdinaryObjectCreate(%AsyncGenerator.prototype%).
               1. Perform DefinePropertyOrThrow(_F_, *"prototype"*, PropertyDescriptor { [[Value]]: _prototype_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *false* }).
             1. Else if _kind_ is ~normal~, perform MakeConstructor(_F_).
-            1. NOTE: Async functions are not constructable and do not have a [[Construct]] internal method or a *"prototype"* property.
+            1. NOTE: Functions whose _kind_ is ~async~ are not constructable and do not have a [[Construct]] internal method nor a *"prototype"* property.
```

Comment:
This should remain an "or". "nor" tends to be paired with "neither", or only used when preceding a verb phrase. In this case we have two noun phrases.